wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Content/Rich Content/ConversationImageMessageCell.swift

```diff
@@ -70,24 +68,26 @@ final class ConversationImageMessageCell: UIView,
     }
 
     private func configureView() {
-        containerView.translatesAutoresizingMaskIntoConstraints = false
-
         containerView.layer.cornerRadius = 12
         containerView.layer.borderWidth = 1
         containerView.layer.masksToBounds = true
         containerView.backgroundColor = SemanticColors.View.backgroundCollectionCell
         containerView.layer.borderColor = SemanticColors.View.backgroundSeparatorCell.cgColor
 
+        containerView.translatesAutoresizingMaskIntoConstraints = false
         addSubview(containerView)
     }
 
     private func createConstraints() {
-        containerView.translatesAutoresizingMaskIntoConstraints = false
+        let margins = conversationHorizontalMargins
 
-        let leading = containerView.leadingAnchor.constraint(equalTo: leadingAnchor)
-        let trailing = containerView.trailingAnchor.constraint(lessThanOrEqualTo: trailingAnchor)
```

Comment:
Was the `lessThanOrEqualTo` constraint correct this way or should it have been inverted:
```
trailingAnchor.constraint(lessThanOrEqualTo: containerView.trailingAnchor)
```